RxCocoa/Common/DelegateProxyType.swift

```diff
@@ -86,56 +86,98 @@ and extend it
 
 */
 public protocol DelegateProxyType : AnyObject {
-    /// DelegateProxy factory
+    associatedtype ParentObject: AnyObject
+    associatedtype Delegate: AnyObject
+    
+    /// DelegateProxy factory. Use 'DelegateProxyFactory.sharedFactory'.
     static var factory: DelegateProxyFactory { get }
 
+    /// Creates new proxy for target object.
+    /// Should not call this function directory, use 'DelegateProxy.proxyForObject'
+    static func createProxy(for object: AnyObject) -> AnyObject
+
+    /// Store DelegateProxy subclass to factory.
+    /// When make 'RxXXXDelegateProxy' subclass, call 'RxXXXDelegateProxySubclass.register()' 1 time.
```

Comment:
Maybe we can use `Rx*DelegateProxy` or some other wildcard instead of `'RxXXXDelegateProxy`. Seems a bit nicer 😄 .

I think the same would apply to other parts of docs.